app/cdap/components/InlayDrawerWidget/InlayDrawerWidget.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { withInfo } from '@storybook/addon-info';
+import { storiesOf } from '@storybook/react';
+import InlayDrawerWidget, { IMenuItem } from 'components/InlayDrawerWidget';
+import T from 'i18n-react';
+import React, { useState } from 'react';
+
+export const PREFIX = 'features.WranglerNewUI.Drawer';
+
+storiesOf('InlayDrawerWidget', module).add(
+  'Default Inlay Drawer',
+  withInfo({
+    text: 'Default Drawer',
+  })(() => {
+    const [open, setOpen] = useState(true);
```

Comment:
This could use `boolean` from storybook knobs instead of state. Then the user could close and re-open the drawer.

Reply:
@njbriggs 
updated the story book to have knobs and actions
plz review code and confirm